spopt/tests/test_c_p_median.py

```diff
@@ -15,6 +15,11 @@
 import pytest
 import os
 
+from packaging.version import Version
+
+# see gh:spopt#437
+GPD_GT_0142 = Version(geopandas.__version__) > Version("0.14.2")
```

Comment:
This is incorrect. Potential 0.14.3 will not have `union_all`. I would just check for 1.0 here and live with the warning temporarily, knowing it is resolved.